.travis.yml

```diff
@@ -203,6 +219,26 @@ jobs:
         - tar xvfz generate_secret-windows-amd64-${APP_VERSION}.tar.gz -C /tmp
         - /tmp/generate_secret.exe
 
+      after_failure:
+        - SLACK_TEXT="${APP_VERSION} $TRAVIS_OS_NAME assets failed verification." send_to_slack.sh
+
```

Comment:
if any of the previous stages fail, it'll never get to this last stage. so if it got this far, then that means we're all set.

Reply:
If none of the previous tests fail, then why can't we publish via the API as opposed to manually clicking publish? If the tests do fail then the draft is still there to manually verify and publish, which I believe is the motivation for the initial draft release.

Reply:
agree.. adding automatic publishing once all the testing stages pass (including the final verification ones)